src/Apps/CollectorProfile/Routes/Saves2/CollectorProfileSaves2Route.tsx

```diff
@@ -135,7 +135,7 @@ export const CollectorProfileSaves2RouteFragmentContainer = createFragmentContai
   {
     me: graphql`
       fragment CollectorProfileSaves2Route_me on Me {
-        allSavesArtworkList: collection(id: "saved-artwork") {
+        savedArtworksArtworkList: collection(id: "saved-artwork") {
```

Comment:
**suggestion**: Maybe to avoid further churn we can call this "defaultCollection" or "standardCollection"?

Reply:
we recently made a small agreement on the use of uniform terminology ([link 1](https://artsy.slack.com/archives/C9SATFLUU/p1680012458469089?thread_ts=1680012130.077609&cid=C9SATFLUU), [link 2](https://www.notion.so/artsy/Artworks-list-I-think-we-ve-got-a-confusing-mix-of-terms-in-our-codebase-for-Artwork-Lists-collec-03de616986ab4cd4a839ec6aed989d3d))

@anandaroop what do you think about the suggested changes by Devon?

Reply:
I'd be fine with the change here, or `defaultArtworkList` as well. 

I like the idea of sticking to the pattern where Relay-and-upstream continues to use the "connection" terminology, whereas client-side code aligns on "artwork list", [using the Relay query as the point where we "translate."](https://github.com/artsy/force/pull/12171#discussion_r1153273394)

Reply:
I'm afraid this will add more confusion. for example, somewhere there will be a mention of "default artwork list", somewhere "saved artworks list". for this reason, I decided to leave only the current name.

In any case, thank you all very much for your participation and discussions 🙏